datadog_checks_base/datadog_checks/base/checks/network.py

```diff
@@ -11,46 +11,9 @@ class Status:
     UP = "UP"
 
 
-class NetworkCheck(AgentCheck):
```

Comment:
It's also used by integrations-extras here: https://github.com/DataDog/integrations-extras/blob/cffb4808ddb6159edc8d2bcddb3138597ec7723d/snmpwalk/datadog_checks/snmpwalk/snmpwalk.py#L15